lib/logstash/inputs/jdbc.rb

```diff
@@ -329,6 +317,7 @@ def set_value_tracker(instance)
   end
 
   def run(queue)
+    load_driver
```

Comment:
This is another part of the reverted commit, and I'm not entirely sure why it was that way before.

I _do_ know that failures in `#register` will prevent pipeline instantiation, while failures in `#run` will occur after the pipeline has been initialized, and that it would be better to continue aborting early if we cannot load the driver. 

I'll make a stab at bouncing it back up into `#register`, and see what ramifications that has on tests.